frontend/src/api/index.ts

```diff
@@ -10,3 +10,13 @@ export async function loginUser(token: string): Promise<User> {
   const user = res.data as User;
   return user;
 }
+
+export async function updateUser(token: string, userInfo: any): Promise<User> {
```

Comment:
`userInfo` shouldn't be an `any` because it is clear what we expect. (In general, when using Typescript, we want to avoid `any` as much as possible as it foregoes the type safety benefits of Typescript). In frontend/src/types.ts we should define a UserDetails type. 

When we define the state in [UpdateUser.tsx](https://github.com/Siyer2/uni-connector/blob/0b439765e64fd259efa1320f0b7e9c38365737a9/frontend/src/pages/UpdateUser.tsx#L23), we should use this UserDetails type. Here in the `updateUser` function, we should also use the UserDetails type.

Also, rather than using UserInfo in some places and UserDetails in other places, we should standardise this and only use 1 everywhere. It will save from confusion in the future (particularly if there's new developers looking at this code).